src/components/Lobby/GameView/PlayerView/PlayerView.tsx

```diff
@@ -136,7 +136,7 @@ export default function PlayerView(props: PlayerViewProps): JSX.Element {
 					<div className={styles['col-gap']}>
 						<div className="flex flex-row justify-between mb-4">
 							<div className="flex-none md:-top-11 z-10">
-								<Countdown limitDate={dayjs(props.lobby.game.limitDate)} onFinish={nextDrawing} />
+								<Countdown limitDate={dayjs(props.lobby.game.limitDate)} />
```

Comment:
plus besoin de l'event onFinish ?

Reply:
Indeed, because now this event is handled by the server instead of by the Countdown, so no need to call it anymore via onFinish!